base/int.jl

```diff
@@ -57,18 +57,28 @@ typemin(typeof(x))`, `abs(x) == x`, not `-x` as might be expected.
 abs(x::Signed) = flipsign(x,x)
 
 """
+    Base.checked_abs(x)
+
+The absolute value of `x`, with overflow error trapping where applicable.
+For types for which no overflow can happen during `abs`, this is equivalent
+to `abs(x)`.
+"""
+function checked_abs end
```

Comment:
I wanted a description that applies both to signed types, as well as to `Unsigned`, `BigInt`, and to any other unsigned type that could be added a method elsewhere. So it sounded more logical to attach it to the function, and only give type-specific details when needed (e.g. for `Signed` below).

But I don't really understand how this works. Doesn't the method to which you attach the docstring make any difference? How are the docstrings sorted? Cf. https://github.com/JuliaLang/julia/issues/13911 I just filed today.